### PR TITLE
[Editor] Correctly save a non-ascii alt text

### PR DESCRIPTION
**Pull request by calixteman**

### CODE REVIEW

File: src/core/struct_tree.js [hunk 1]
```diff
@@ -281,6 +282,12 @@ class StructTreeRoot {
     }
   }
 
+  static convertString(str) {
+    return isAscii(str)
+      ? str
+      : stringToUTF16String(str, /* bigEndian = */ true);
+  }
```
Comment (Snuffleupagus) [inline src/core/struct_tree.js:289]: The same code seems to be duplicated *multiple* times already in https://github.com/mozilla/pdf.js/blob/master/src/core/annotation.js, so can we please add a new helper function (placed in `/src/core/core_utils.js`) and use that everywhere instead?

Comment (calixteman) [reply src/core/struct_tree.js:289]: Definitely, you've a way better memory than me... I'm a bit jealous :).

**Review by Snuffleupagus — APPROVED** [review]
r=me, thank you!

**Pull request merged**